Snakefile

```diff
@@ -81,7 +81,7 @@ rule create_scenarios:
     output:
         config["run"]["scenarios"]["file"],
     conda:
-        "envs/retrieve.yaml"
+        conda_env_provider()
```

Comment:
Maybe it makes sense to allow an argument here to a file "envs/retrieve.yaml" or similar/its pinned version?

I think there is some untapped potential to use rule-specific environments rather than one large environment.